sdk/src/Core/Amazon.Runtime/IClientConfig.cs

```diff
@@ -55,6 +57,14 @@ public partial interface IClientConfig
         /// </summary>
         Profile Profile { get; }
 
+        /// <summary>
+        /// Gets the configuration for identity resolvers used by the service client, which manages
+        /// the resolvers used to obtain various types of identities.
```

Comment:
```suggestion
        /// Gets the configuration for the identity resolvers configured in the service client.
```

I think the documentation could be improved, but I don't have a better suggestion at the moment (looks like Java ended up not using `*Configuration` at all: https://github.com/aws/aws-sdk-java-v2/pull/4423).

Reply:
I did see that and it took me a while to figure out that the `IdentityResolvers` is the `IdentityResolverConfiguration` mentioned in the SRA, so I would prefer to use the SRA names to make it easier to find what it means in the SRA unless we have a good reason not to do so.

Reply:
Oh, for sure. I wasn't suggesting renaming the type, just mentioning we can probably improve the documentation for this property (since it's visible in the config and may not be clear to customers at first).